app/editor/src/index.scss

```diff
@@ -1,7 +1,10 @@
 @import './css/_colors';
 
-body {
+html,
+body,
+main {
   margin: 0;
+  height: 100%;
```

Comment:
It appears that we've made a mess and it's near impossible to figure out how to expand.  Examples highlight the need of 100% height with the use of flexbox.